docs/reference/settings/ml-settings.asciidoc

```diff
@@ -67,7 +67,8 @@ inference model cache. The TTL is calculated from last access. The `inference`
 processor attempts to load the model from cache. If the `inference` processor
 does not receive any documents for the duration of the TTL, the referenced model
 is flagged for eviction from the cache. If a document is processed later, the
-model is again loaded into the cache. Defaults to `5m`.
+model is again loaded into the cache. To update this setting in {ess}, see 
```

Comment:
> xpack.ml.inference_model.time_to_live

This description mentions only inference processors.  Does it also affect inference aggregation?

Reply:
Yes inference aggregation is a user of this cache so the same setting applies. It's better not to describe the caching action in terms of the inference processor receiving documents as there could be other users of the cache.  Can you re-phase it something like this pls:

"
The time to live (TTL) for models in the inference model cache. The TTL is calculated from last access. 
Users of the cache such as the `inference` processor or the `inference` aggregator cause a model to 
be cached on first use and reset the TTL on every use. If a cached model is not accessed for the duration 
of the TTL, the referenced model is flagged for eviction from the cache. 
If a document is processed later, the model is again loaded into the cache. Defaults to `5m`.
"

Reply:
Thanks!  I updated this text in https://github.com/elastic/elasticsearch/pull/69947/commits/d04db2f4df24e34e4f56b66b448625b7e8667cb0. I also added the "trained model" terminology, since that's the term we use elsewhere in the docs.